l10n_es_intrastat_report/wizards/l10n_es_intrastat_code_import.py

```diff
@@ -4,20 +4,18 @@
 
 import os
 
-from odoo import _, exceptions, models, tools
-
-try:
-    import xlrd
-except ImportError:
-    xlrd = None
+import xlrd
```

Comment:
add xlrd in __manifiest__ ?
https://github.com/OCA/l10n-spain/blob/13.0/l10n_es_intrastat_report/__manifest__.py

Reply:
It's an Odoo dependency, so it should be already in the Python path.

Reply:
ok!

Reply:
Remove the requested changes then